app/javascript/components/timeline-options/timeline-chart.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { LineChart } from '@carbon/charts-react';
+import { timelineUiOptions } from './timeline-helper';
+
+const TimelineChart = ({ data, title, buildTableData }) => {
+  const chartRef = useRef(null);
+  const chartOnClick = ({ detail }) => {
+    buildTableData(detail.datum);
+  };
```

Comment:
just a suggestion. this can be written in a single line

`const chartOnClick = ({ detail }) => buildTableData(detail.datum);`